components/calendar/CalendarWeekly.tsx

```diff
@@ -133,11 +133,11 @@ const CalendarWeekly: React.FC<CalendarProps> = ({
 
       const calColor =
         event.title === "CCV Office Hours"
-          ? "bg-50 hover:bg-300"
+          ? "bg-keppel-100 hover:bg-keppel-300"
           : event.title ===
               "COBRE CBC Computational Biology Walk-in Office hours"
-            ? "bg-50 hover:bg-300"
-            : "bg-keppel-50 hover:bg-keppel-500"
+            ? "bg-keppel-100 hover:bg-keppel-300"
+            : "bg-keppel-100 hover:bg-keppel-300"
```

Comment:
I'm not really sure why we have this there in the first place. I guess we wanted the option to colorize office hour events differently? This seems like a pain to maintain if the titles change.

Reply:
that's fine! if they were all the same color it's chill unless people want different colors